File: jScrap/src/main/java/com/sorception/jscrap/api/AcceptedController.java
package com.sorception.jscrap.api;

import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RequestMethod;
import org.springframework.web.bind.annotation.ResponseBody;

import com.sorception.jscrap.entities.OfferEntity;
import com.sorception.jscrap.services.OfferService;

@Controller
@RequestMapping("/api/accepted")
public class AcceptedController {
	@Autowired
	private OfferService offerService;
	
	@RequestMapping(value = "", method = RequestMethod.GET)
	@ResponseBody
	public List<OfferEntity> getOffers() {
		return offerService.getAcceptedOffers();
	}
}

File: Scrap.NET/ScrapWeb/Controllers/AcceptedController.cs
using ScrapWeb.Entities;
using ScrapWeb.Services;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Net;
using System.Net.Http;
using System.Web.Http;

namespace ScrapWeb.Controllers
{
    [Authorize]
    public class AcceptedController : ApiController
    {
        private OfferService offerService;

        AcceptedController() 
        {
            offerService = new OfferService();
        }

        // GET api/<controller>
        public IEnumerable<OfferEntity> Get()
        {
            return offerService.getAccepted();
        }
    }
}